addon-test-support/-private/ember-exam-qunit-test-loader.js

```diff
@@ -126,7 +126,9 @@ export default class EmberExamQUnitTestLoader extends TestLoader {
     this._qunit.moduleDone((metadata) => {
       // testem:module-done-metadata is sent to server to keep track of test module details.
       // 'metadata' contains module name, total number of assertion ran in the module, and module runtime.
-      this._testem.emit('testem:module-done-metadata', metadata);
+      if (typeof this._testem !== 'undefined' && this._testem !== null) {
```

Comment:
When does this occur, and could a test be written?

Reply:
just added a test to test to validate testLoader loads modules when testem is not available :)